Assets/PatchKit Patcher/Scripts/AppUpdater/AppUpdaterConfiguration.cs

```diff
@@ -3,10 +3,12 @@
 namespace PatchKit.Unity.Patcher.AppUpdater
 {
     [Serializable]
-    public struct AppUpdaterConfiguration
+    public class AppUpdaterConfiguration
```

Comment:
Why class instead of struct?

Reply:
struct cannot have default field initializers and I wont leave config with sizeThreshold that would be 0 by default in most cases.

Reply:
Ok - you're right, Unity won't allow us to set some default value here with struct. Could you confirm that class solution works in Unity (default value is set)?

Reply:
Yes, I see this value preserved in inspector.